### ISSUE TITLE
Accessibility: Consider increasing minimum page thumbnail size

### ISSUE THREAD

**Issue by miina** [GoogleForCreators/web-stories-wp#1099]
## Feature Description
Currently, the minimum page thumbnail size seems to be so small that it's not really accessible anymore -- it would be good to consider setting a minimum size so that it would be easier to understand what's in the preview.

Example from a 13'' screen:
<img width="422" alt="Screenshot 2020-04-08 at 11 08 17" src="https://user-images.githubusercontent.com/3294597/78766776-fb311600-7989-11ea-9df6-83be12a6f52e.png">

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

1. Minimum Height should be 52 px.

**Comment by pbakaus** [comment]
This honestly seems like a bug. It should never be rendered that tiny. Agree. At that resolution, they should become blue bars only.

**Comment by dvoytenko** [comment]
Can we just quickly pick a minimal height value here? Maybe `60px`?

**Comment by pbakaus** [comment]
Let's make it `52` for now.
